### ISSUE TITLE
Various components: add/enhance keyboard support

### ISSUE THREAD

**Issue by mindrones** [nestauk/svizzle#630]
Lint fails on Choropleth and Histogram with error:

`A11y: visible, non-interactive elements with an on:click event must be accompanied by an on:keydown, on:keyup, or on:keypress event  a11y-click-events-have-key-events`
